Add window-leveling to object classification, labeling and counting

paulhfu opened the pull request:
fixes #1754
Window-Leveling should be in every labeling-gui available.

k-dominik commented:
Very nice, checked it in object classification, tracking, counting!

Since all the Gui classes you edited here derive from `LabelingGui`, I still wonder whether we could somehow get away with having something like a `_setupLayers` setting up the window-leveling bit for the raw data in `LabelingGui` and just call it from the respective child classes `setupLayers`; what do you think @paulhfu

paulhfu commented:
Thanks for the comment @k-dominik !
Not sure if I get this right. But, LabelingGui already has a setupLayers which is called in the respective child's setupLayers. It also already contains all setup for the window-leveling. This setup is just never executed since on instantiation of  LabelingGui no rawInputSlot object is given to LabelingGui which is required for the setup. 
It would certainly make more sense to set everything in the base class. I will do that with the next commit.

k-dominik commented:
hey @paulhfu ,

the commits look a bit odd (duplicate commit messages), maybe we should try to resolve this together.

k-dominik commented on ilastik/applets/counting/countingGui.py:
```diff
@@ -647,10 +651,19 @@ def setupLayers(self):
+            for i in range(len(layers)):
+                if layers[i].name == "Raw Input":
+                    inputLayer = layers[i]
+                    layers[i], layers[-1] = layers[-1], layers[i]
```
Hey, for things like these, where you swap stuff, it would be really helpful to add a line of comment on why (not obvious to me)

k-dominik commented on ilastik/applets/labeling/labelingGui.py:
```diff
@@ -48,6 +48,8 @@
 
 from ilastik.applets.labeling.labelingImport import import_labeling_layer
 
+from volumina.api import \
```
I don't know if you committed some auto changes here in this file in general (by some linter)... But our current agreement is not to do such changes because it's easier to navigate file history.
In any case I would also _always_ opt for parentheses for multi-line imports such as
```python
from volumina.api import (
    GrayscaleLayer, ColortableLayer, LazyflowSinkSource
)
```

paulhfu commented:
Hi @k-dominik,
Good that you noticed that. When i moved the init of the raw layer to the baseclass, i changed the order of layers in the layerstack. In counting there where obvious problems which is why i reordered it there. But the update problem in the other workflows also result from that. So, either i apply the reordering which i did in counting also to the other workflows or i find out why the layer stack has to be in this static order and change it. We can just talk Monday about that.

k-dominik commented:
@paulhfu, so should I give this another review? Really want to merge it soon :)

paulhfu commented:
@k-dominik, sure go ahead.

paulhfu commented on ilastik/applets/objectClassification/labelingDrawer.ui:
```diff
@@ -153,7 +153,21 @@
+		</widget>
+	   </item>
        <item>
+        <widget class="QToolButton" name="thresToolButton">
```
You're totally right. I removed it.

paulhfu commented:
Hey @k-dominik, rebase is done
